pg_search/sql/pg_search--0.10.3--0.10.4.sql

```diff
@@ -169,4 +169,9 @@ CREATE  FUNCTION "index_info"(
                 )
     STRICT
     LANGUAGE c /* Rust */
-AS 'MODULE_PATHNAME', 'index_info_wrapper';
\ No newline at end of file
+AS 'MODULE_PATHNAME', 'index_info_wrapper';
```

Comment:
Can you rename this file to 0.10.3--0.11.0 please? So we don't forget

Reply:
I don't want to do that in this PR, it's not relevant to the changes I've made.